resource/select_gpu_devices.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+import re
+import dataiku
+from constants import GPU_CONFIGURATION
+from safe_logger import SafeLogger
+
+
+logger = SafeLogger("Forecast plugin")
+
+
+def do(payload, config, plugin_config, inputs):
+    """ Retrieve a list of available GPU devices. """
+    choices = []
+
+    if payload.get("parameterName") == "gpu_devices":
+        try:
+            import mxnet as mx
+        except OSError as cuda_error:  # error when importing mxnet
```

Comment:
Can we target a more specific error - see https://github.com/apache/incubator-mxnet/blob/1c3fcad78caa7e58f81045d6128435a59222dfdd/python/mxnet/base.py#L86